Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -876,9 +876,7 @@ bool CStaticFunctionDefinitions::SetElementData(CElement* pElement, const char*
     assert(szName);
     assert(strlen(szName) <= MAX_CUSTOMDATA_NAME_LENGTH);
 
-    bool          bIsSynced;
-    CLuaArgument* pCurrentVariable = pElement->GetCustomData(szName, false, &bIsSynced);
-    if (!pCurrentVariable || *pCurrentVariable != Variable || bIsSynced != bSynchronize)
+    if (pElement->SetCustomData(szName, Variable, bSynchronize))

```

Comment:
this method needs modification to fix the capitalisation of variable (I'm aware it was like that before but it should be fixed nonetheless)

Reply:
Alright, will fix it tomororw, gotta sleep

Reply:
Should we keep the capitalisation of Variable in the sturct?

Reply:
Or I can change that to lowercase as well?

Reply:
actually, nevermind, we can address this another time. it was probably done that way for a reason although i can't think of one why